backend/src/API/mailAPI.ts

```diff
@@ -50,9 +51,65 @@ const emailAdmins = async (req: Request, subject: string, text: string) => {
   });
 };
 
+const emailMember = async (req: Request, member: IdolMember, subject: string, text: string) => {
+  const url = getSendMailURL(req);
+  const idToken = req.headers['auth-token'] as string;
+  const requestBody = {
+    subject,
+    text
+  };
+
+  return axios.post(
+    url,
+    { ...requestBody, to: member.email },
+    { headers: { 'auth-token': idToken } }
+  );
+};
+
 export const sendMemberUpdateNotifications = async (req: Request): Promise<Promise<void>[]> => {
   const subject = 'IDOL Member Profile Change';
   const text =
     'Hey! A DTI member has updated their profile on IDOL. Please visit https://idol.cornelldti.org/admin/member-review to review the changes.';
   return emailAdmins(req, subject, text);
 };
+
+export const sendTECReminder = async (req: Request, member: IdolMember): Promise<AxiosResponse> => {
+  const subject = 'TEC Reminder';
+  const allEvents = getAllTeamEvents(req.body);
+  const futureEvents = (await allEvents).filter((event) => {
+    const eventDate = new Date(event.date);
+    const todayDate = new Date();
+    return eventDate >= todayDate;
+  });
+  const memberEvents = getTeamEventAttendanceByUser(member);
+  let approvedCount = 0;
+  let pendingCount = 0;
+  (await memberEvents).forEach((event) => {
+    if (event.status === 'approved') {
+      approvedCount += 1;
+    }
+    if (event.status === 'pending') {
+      pendingCount += 1;
+    }
+  });
+
+  const text =
+    `Hey! You currently have ${approvedCount} team event credits approved and ${pendingCount} team event credits pending this semester. ` +
+    `This is a reminder to get at least ${
+      member.role === 'lead' ? '6' : '3'
+    } team events credits by the end of the semester.\n` +
+    `\n${
+      futureEvents.length === 0
+        ? 'There are currently no upcoming team events, but check IDOL soon for updates.'
```

Comment:
> check IDOL soon for updates
Thoughts on changing this to `There are currently no upcoming team events listed on IDOL, but check the #team-events channel for upcoming team events`.